build/tasks/dotnet-buildtools.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <DotNetBuildToolsVersion>2.1.0-prerelease-02430-04</DotNetBuildToolsVersion>
```

Comment:
Can we put this in dependencies.props as `DevDependency_DotNetBuildToolsVersion`? Same for the other package versions listed in RepoTasks.